classes/order/OrderReturn.php

```diff
@@ -62,8 +62,15 @@ class OrderReturnCore extends ObjectModel
         ],
     ];
 
-    public function addReturnDetail($order_detail_list, $product_qty_list, $customization_ids, $customization_qty_input)
+    public function addReturnDetail($order_detail_list, $product_qty_list)
     {
+        if (func_num_args() > 2) {
+            @trigger_error(
```

Comment:
Removing arguments is BC break, I suggest to do it like that to deprecate it:
```suggestion
    public function addReturnDetail($order_detail_list, $product_qty_list, $customization_ids = null, $customization_qty_input = null)
    {
        if ($customization_ids !== null || $customization_qty_input !== null) {
            @trigger_error(
```

Reply:
Resolved in https://github.com/PrestaShop/PrestaShop/pull/12422/commits/ba5f7ae6de13acbf71a11cd67c5a7e0e9cd912b4